static/docs/user-guide/using-docker.md

```diff
@@ -0,0 +1,108 @@
+# Using Docker
+
+Currently we don't provide any official Docker image, however you can easily
```

Comment:
If we have the Docker file ready, why don't we create an Image?

Reply:
@shcheklein , being honest, I think that having an official image would bring more harm than benefits (maintenance burden, releases, etc.).

This is only an **example** for the ones who are completely lost at the topic. It is not intended to serve as a **reference** for best practices on writing Docker images using `dvc`.

Reply:
I don't think there's much point documenting anything other than a reference for best practices... :confused:

Reply:
publishing a complete Dockerfile like this is also a burden - we'll need to keep it updated, right?

if we see some value in Docker I would create that image (it's quite straightforward to make it part of CI) or put a Dockerfile under Git and make a doc on how to start using it. It's quite strange to publish a complete Dockerfile this way.

@casperdcl not sure I got what way your argument is going :)

Reply:
Yes trying to explain my ethos here: https://github.com/iterative/dvc.org/pull/811#issuecomment-557819169